auslib/web/common/history.py

```diff
@@ -0,0 +1,107 @@
+import json
```

Comment:
I think that makes a lot of sense, I was just making sure I understood :). It's probably worth adding a comment and/or filing a follow-up bug about converting it (better not to do it here - this PR is big enough already).